printer_mods/Tehtsuo/Case_Fan_Splicer/Readme.md

```diff
@@ -0,0 +1,38 @@
+# Case Fan Splicer
+
+**Purpose:**
+
+An alternative to a PCB for wiring 2 pin JST-XH connectors in parallel.
+
+![Diagram](./Diagram.png)
+
+**Why:**
+
+Didn't feel like waiting for a PCB to be made and shipped to me, and I'd likely end up making a container for the PCB to go into anyways.
+
+![Diagram](./Installed.png)
+
+**BOM:**
+
+* 2x m3 x 12-ish
+* 2x Your choice of m3 hammer nut or m3roll in t nut
```

Comment:
M3 should be capitalized throughout this file (yes, it's nitpicky, sorry)
you're missing some hyphens and spaces here. `M3 hammerhead or roll-in t-nut`